storm-core/src/storm.thrift

```diff
@@ -174,12 +174,14 @@ struct BoltStats {
   3: required map<string, map<GlobalStreamId, double>> process_ms_avg;
   4: required map<string, map<GlobalStreamId, i64>> executed;  
   5: required map<string, map<GlobalStreamId, double>> execute_ms_avg;
+  6: required map<string, map<GlobalStreamId, double>> deserialize_time;
 }
 
 struct SpoutStats {
   1: required map<string, map<string, i64>> acked;
   2: required map<string, map<string, i64>> failed;
   3: required map<string, map<string, double>> complete_ms_avg;
+  4: required map<string, map<string, double>> deserialize_time;
```

Comment:
This is still marked as required, if we want a rolling upgrade this needs to be optional, possibly with a reasonable default, and every time we access it, mostly for reads, we need to check to see if it is set.